vulnerabilities/models.py

```diff
@@ -152,6 +148,14 @@ def with_package_counts(self):
         )
 
 
+class VulnerabilityStatusType(models.IntegerChoices):
+    """List of vulnerability statuses."""
+
+    PUBLISHED = 1, "published"
```

Comment:
The code in https://github.com/nexB/vulnerablecode/pull/1232/files#diff-3e8fc96d993d0bd8585642f68239732a30a19b7ec100a36ee5d21e209625ff81R120 seem redundant. Could we have a single label, either as "published" or "Published" but not both?